__tests__/utils/test-case.ts

```diff
@@ -83,6 +83,7 @@ export class TestCase {
     __resetSingletons();
 
     const app = await NestFactory.create(this.module, this.adapter, {
+      abortOnError: false,
```

Comment:
what's the reason for this? not sure about that

Reply:
@iamolegga Because Nest.js calls `process.exit` if error is thrown during initialization phase. Therefore, errors thrown during testing will kill entire testing process (often without logs), not just fail a test.

Reply:
ah, ok then